packages/@uppy/dashboard/src/components/AddFiles.js

```diff
@@ -125,6 +125,7 @@ class AddFiles extends Component {
           className="uppy-u-reset uppy-c-btn uppy-DashboardTab-btn"
           role="tab"
           tabIndex={0}
+          data-cy={acquirer.id}
```

Comment:
Is this from a bad merge commit?

Reply:
You can read about it here: https://docs.cypress.io/guides/references/best-practices#Selecting-Elements